### ISSUE TITLE
Stylistic improvements

### ISSUE THREAD

**Issue by Cimbali** [Cimbali/CleanLinks#87]
Possibly inspired from other similar add-ons, with small fonts and symbol-based buttons rather than text-based, with a tighter layout.

**Comment by Cimbali** [comment]
I’m looking at basing the buttons on the [octicons](https://octicons.github.com/). Not sure if the shield or list conveys the “whitelist/rules” better:
<img height="50" width="50" alt="list-1" src="https://user-images.githubusercontent.com/6126377/77005566-4c3c8400-6961-11ea-93d7-696ef73fadf8.png" /> <img height="50" width="50" alt="list-check" src="https://user-images.githubusercontent.com/6126377/77005571-4d6db100-6961-11ea-8bc7-165928800acc.png" /> <img height="50" width="50" alt="list-edit" src="https://user-images.githubusercontent.com/6126377/77005573-4d6db100-6961-11ea-8d7e-7f7ec9ef36ee.png" /> <img height="50" width="50" alt="list-x" src="https://user-images.githubusercontent.com/6126377/77005574-4e064780-6961-11ea-91eb-587c26a3b0fe.png" />
<img height="50" width="50" alt="shield-1" src="https://user-images.githubusercontent.com/6126377/77005575-4e9ede00-6961-11ea-9add-8f2c44afe2e7.png" /> <img height="50" width="50" alt="shield-check" src="https://user-images.githubusercontent.com/6126377/77005577-4f377480-6961-11ea-851b-eb3f0b12bae9.png" /> <img height="50" width="50" alt="shield-edit" src="https://user-images.githubusercontent.com/6126377/77005578-4f377480-6961-11ea-964d-a31aace1b907.png" /> <img height="50" width="50" alt="shield-x" src="https://user-images.githubusercontent.com/6126377/77005579-4fd00b00-6961-11ea-8d48-fcaa45480272.png" />

The buttons being for whitelist once / add to whitelist / remove from whitelist / edit whitelist.

**Comment by Cimbali** [comment]
Some changes are now in master. Currently the editing of rules looks like this:

![WhatsApp Image 2020-03-21 at 23 47 07](https://user-images.githubusercontent.com/6126377/77238456-721b9000-6bd0-11ea-93fd-7ad837dfb9a0.jpeg)

The popup looks something like this when opened from the toolbar:

![Screenshot_20200322_005319](https://user-images.githubusercontent.com/6126377/77239205-2e2c8900-6bd8-11ea-9d70-974b85da8c14.png)

And something like this when opened from the overflow menu:

![Screenshot_20200322_005643](https://user-images.githubusercontent.com/6126377/77239207-2ff64c80-6bd8-11ea-9e2c-618ff592bffb.png)


---

Here are what the preferences and popup respectively look like on Android and in French:
<img src="https://user-images.githubusercontent.com/6126377/77238934-7f874900-6bd5-11ea-8cf2-d27583d21038.png" width="300" /> <img src="https://user-images.githubusercontent.com/6126377/77238937-81510c80-6bd5-11ea-907f-13a6f2f8e4b0.png" width="300" />

Basically less spacing and bigger buttons, more vertical layouts, the rule selector is rather small as it is edited as a popup.

**Comment by Cimbali** [comment]
Any feedback/suggestions welcome. As you can see the wider popup is useful to look at very long links (with wrapping for selected links). If that’s not desirable, maybe we can use a setting with the preferred popup size.

I think these blue and red backgrounds in the history list need to go as well.

**Comment by Cimbali** [comment]
Bit more subtle colours and marking of selected entry:

![Screenshot_20200322_021847](https://user-images.githubusercontent.com/6126377/77240240-f62b4300-6be3-11ea-94da-38e1ef1f91b5.png)

**Comment by Mike-Reddick** [comment]
a optional dark theme can be welcome

**Comment by Rtizer-9** [comment]
I think redirections which are blocked are more visible in the red background like earlier (just like all pics here except last one) as compared to the current implementation of cross button.

One more thing: earlier entries were clickable but now it doesn't change color so it creates a confusion whether it's selected or not (for adding to whitelist etc)

**Comment by Cimbali** [comment]
Alright so some colouring is back, but a little more subtle than previously. Hopefully it both looks OK and is not too confusing now.
